bin/release.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pipenv run python manage.py migrate
```

Comment:
Suggestion: add the BJCP management command here and loading schedule fixtures so we can guarantee the database gets seeded

Reply:
The only issue is that reloading those things can go poorly, and Heroku is going to want to rerun this script _a lot_ (basically any time a config or source changes).  I guess we add some logic to the import script to prevent double-import?

Reply:
True. I'm good with leaving them manual for now.

Reply:
Before I merge, do you need to drop `pipenv run` here like you did in `entry-point.sh`?

Reply:
It seems to actually want it here, not sure why.